knotx-core/src/main/java/io/knotx/util/DataObjectsUtil.java

```diff
@@ -62,11 +62,11 @@ public static boolean equalsBody(io.vertx.core.buffer.Buffer self,
   }
 
   /**
-   * Method computing hashCode of the give Multimap.
-   * Multimap is treat as list of entires, key(String), value(List of Strings)
+   * Method computing hashCode of the give MultiMap.
+   * MultiMap is treat as list of entries, key(String), value(List of Strings)
```

Comment:
> MultiMap is treat as list of entries, key(String), value(List of Strings)

Something wrong with the grammar. The passive form of _treat_ is _treated_. How about:

> MultiMap is treated as a list of key-value pairs where each key is a String and the corresponding value is a List of Strings.